packages/automerge-repo-react-hooks/src/RepoContext.ts

```diff
@@ -0,0 +1,4 @@
+import { Repo } from "automerge-repo"
```

Comment:
I'd put this in with `useRepo` since that's what it's for and this is otherwise just one line.